src/test/kotlin/no/elhub/auth/features/documents/create/CreateDocumentTest.kt

```diff
@@ -446,3 +465,15 @@ class CreateDocumentTest : BehaviorSpec(), KoinTest {
         }
     }
 }
+
+fun ByteArray.isConformant(): Boolean {
```

Comment:
:100: Very cool, both to have this implemented, but also how it just slots into the existing test criteria